src/encoded/audit/genetic_modification.py

```diff
@@ -0,0 +1,31 @@
+from snovault import (
+    AuditFailure,
+    audit_checker,
+)
+
+# flag genetic modifications with purpose "tagging" that lack genetic modification characterization.
+def audit_tagging_modification(value, system):
+    if value['purpose'] == 'tagging' and\
```

Comment:
Use get for dicts unless we've already determined the key exists.  Also put multi line conditionals in () instead of using the /

Reply:
purpose is a required property in the schema
but i have updated